app/controllers/Controller.scala

```diff
@@ -19,29 +19,56 @@ package controllers
 import config.ViewConfig
 import connectors.des.DesConnector
 import javax.inject.{Inject, Singleton}
+import model.des.{ApprovedInformation, Transaction}
 import play.api.Logger
 import play.api.i18n.Messages
 import play.api.mvc._
+import service.des.DesService
 import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions}
 import uk.gov.hmrc.play.bootstrap.controller.FrontendController
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class Controller @Inject() (mcc: MessagesControllerComponents, implicit val viewConfig: ViewConfig, page1: views.html.page1,
+class Controller @Inject() (mcc: MessagesControllerComponents, implicit val viewConfig: ViewConfig, one_payment: views.html.one_payment,
```

Comment:
We were talking about Views recently on tech and architecture. 
We've decided that it's better to have on Views container with all views which is injected in controllers vs injecting particular views into controllers:

```
class Views @Inject() (
    val one_payment: views.html.one_payment,
//...other pages
)
...
class Controller @Inject() (views: Views) {
 ....
 Ok(views.one_payment(...))
}



```

Reply:
Done